src/components/EmojiPicker/EmojiPickerMenu/index.native.js

```diff
@@ -129,18 +206,55 @@ class EmojiPickerMenu extends Component {
 
     render() {
         return (
-            <View style={styles.emojiPickerContainer}>
+            <View style={[styles.emojiPickerContainer]}>
+                <View style={[styles.pt4, styles.ph4, styles.pb1]}>
+                    <RNTextInput
```

Comment:
Can we use an Expensify `TextInput` here so that we are consistent between the platforms? 
https://github.com/Expensify/App/pull/14532/files#diff-67118ceda8cb66e4c6e8367afc02904c8a06f92ac5cb6e9e0a52a0c26433695fR477

Reply:
Sure